src/Components/CommentAdder.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import * as api from "../api";
+import Error from "./Error";
+
+const CommentAdder = ({ currentUser, setComments, reviewID }) => {
+  const [newComment, setNewComment] = useState("");
+  const [error, setError] = useState(null);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setComments((currComments) => {
+      return [
+        { author: currentUser.currentUser, body: newComment },
+        ...currComments,
+      ];
+    });
+    api
+      .postComment(reviewID, newComment, currentUser.currentUser)
+      .then(() => {
+        setNewComment("");
+      })
+      .catch((err) => {
+        setError({ err });
```

Comment:
Nice that we get the error message now. The comment still appears on the page though, despite the fact the message is saying it hasnt gone well. 
We'd need to remove the comment from the page, as well as displaying some feedback to let the user know its not succeeded; a bit confusing otherwise